src/ucp/wireup/wireup.c

```diff
@@ -292,6 +292,25 @@ ucp_wireup_find_remote_p2p_addr(ucp_ep_h ep, ucp_lane_index_t remote_lane,
     return UCS_ERR_UNREACHABLE;
 }
 
+ucp_lane_index_t
+ucp_wireup_ep_configs_use_same_lane(ucp_ep_config_key_t *ep_config_key,
+                                    ucp_ep_config_key_t *another_ep_config_key,
+                                    ucp_lane_index_t lane)
+{
+    ucp_lane_index_t another_lane;
+
+    for (another_lane = 0; another_lane < another_ep_config_key->num_lanes;
+         ++another_lane) {
+        if (ucp_ep_config_lane_is_equal(ep_config_key,
+                                        another_ep_config_key,
+                                        lane, another_lane, 0)) {
```

Comment:
always `true` because last param is `0`

Reply:
we use this in the UCP EP code:
https://github.com/openucx/ucx/blob/72190be12dc5c798eb415862774ea15dcd53e8b2/src/ucp/core/ucp_ep.c#L1057